README.md

```diff
@@ -18,7 +18,7 @@ npm run build
 A simple example of how to use BSON in the browser:
 
 ```html
-<script src="./dist/bson.js"></script>
+<script src="./dist/bson.bundle.js"></script>
```

Comment:
Can the `README.md` include more documentation about what each `dist` file is?

Reply:
I don't think it's necessary since only 1 is committed. It should be self documenting from package.json.

Reply:
Since I don't fully understand it, I think it would be helpful to document what each bundle is in the readme.

Reply:
Done.